packages/data-stores/src/woocommerce-eligibility/test/actions.ts

```diff
@@ -0,0 +1,32 @@
+import { receiveTransferEligibility, requestTransferEligibility } from '../actions';
+import { TransferEligibility } from '../types';
+
+describe( 'actions', () => {
+	it( 'should return a TRANSFER_ELIGIBILITY_REQUEST action', () => {
+		const expected = {
+			type: 'TRANSFER_ELIGIBILITY_REQUEST',
+		};
+
+		expect( requestTransferEligibility() ).toEqual( expected );
+	} );
+
+	it( 'should return a TRANSFER_ELIGIBILITY_RECEIVE action', () => {
```

Comment:
Noting that tests that only test if an action creator function returns the same object it returns, are generally redundant, and seem to add more noise than actually help ensure functionality works.